src/Access/Common/AccessRightsElement.h

```diff
@@ -79,7 +82,7 @@ struct AccessRightsElement
     /// Resets flags which cannot be granted.
     void eraseNonGrantable();
 
-    bool isEmptyDatabase() const { return !any_database && database.empty(); }
+    bool isEmptyDatabase() const { return database.empty() and !anyDatabase(); }
```

Comment:
Is it the same as `default_database`? I mean is it true that
`isEmptyDatabase() == default_database`?

Reply:
No, it's not

Reply:
Can you give an example of the case when `isEmptyDatabase() == true` then?